backend/clap/src/main/java/zpi/ppea/clap/dtos/AnswerDto.java

```diff
@@ -0,0 +1,17 @@
+package zpi.ppea.clap.dtos;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class AnswerDto {
+    private String description;
```

Comment:
There was a rename in the proto file in the other PR. Let's maybe wait for it to be merged and then align the name in the Clap API response too.